src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -343,7 +343,7 @@ def display_shared_memory_warning(warning_message: str) -> None:
 @nameable_op
 def constant(  # noqa: F811
     tensor: Tensor,
-    shared_memory: bool = False,
+    shared_memory: bool = True,
```

Comment:
It would be good to extend Constant py tests with the default behavior case:
https://github.com/openvinotoolkit/openvino/blob/5cd59b83cca68cecf817eaef65dfda79cb08d835/src/bindings/python/tests/test_graph/test_constant.py#L758-L762